src/simtools/applications/generate_simtel_event_data.py

```diff
@@ -1,12 +1,89 @@
 #!/usr/bin/python3
 
 r"""
-Generate a reduced dataset of event data from simulated files.
+Generate a reduced dataset of event data from output of telescope simulations.
 
-Processes sim_telarray output files (typically of type '.simtel.zst') and creates a
-reduced dataset containing shower information, array-level parameters, and data about
+Processes sim_telarray output files (typically of type '.simtel.zst') and creates
+reduced datasets containing shower information, array-level parameters, and data about
 triggered telescopes.
 
+The output consists of an HDF5 or FITS file containing the following tables:
+
+**FILE_INFO**
+
++-------------------+---------+-----------------------------------------------+
```

Comment:
Not sure about this large table description here: So far we don't describe the formats of the output files in the docstrings and this adds quite a lot of lines. Is it really needed or do we gain anything from this description here?

Reply:
This is for the users guide and I think we need to improve significantly the docstrings (quite a few of our application docstrings are poorly written and don't describe what is happening). Documenting what type of data we are writing to disk is important.